mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -45,14 +46,18 @@
 
 CONTENT_TYPE_CSV = "text/csv"
 CONTENT_TYPE_JSON = "application/json"
-CONTENT_TYPE_JSON_RECORDS_ORIENTED = "application/json; format=pandas-records"
-CONTENT_TYPE_JSON_SPLIT_ORIENTED = "application/json; format=pandas-split"
+CONTENT_TYPE_JSON_RECORDS_ORIENTED = "application/json-pandas-records"
```

Comment:
Is this a backwards-incompatible change? We probably shouldn't break old clients.

Reply:
Is there a way we can test? I can revert the two lines of pandas related changes, however, I don't think it works according to my local test. Please take a look here:

https://github.com/mlflow/mlflow/pull/1479#discussion_r295825703

Reply:
Talked offline with @aarondav , he couldn't reproduce my error. Reverting to
```
CONTENT_TYPE_JSON_RECORDS_ORIENTED = "application/json; format=pandas-records"
CONTENT_TYPE_JSON_SPLIT_ORIENTED = "application/json; format=pandas-split"
```